components/eamxx/src/physics/ml_correction/eamxx_ml_correction_process_interface.cpp

```diff
@@ -69,6 +77,7 @@ void MLCorrection::initialize_impl(const RunType /* run_type */) {
   py_correction = pybind11::module::import("ml_correction");
   ML_model_tq = py_correction.attr("get_ML_model")(m_ML_model_path_tq);
   ML_model_uv = py_correction.attr("get_ML_model")(m_ML_model_path_uv);
+  ML_model_sfc_fluxes = py_correction.attr("get_ML_model")(m_ML_model_path_sfc_fluxes);
   ekat::enable_fpes(fpe_mask);
```

Comment:
I don't see `fpe_mask` being initialized. Perhaps you are missing a line like
```c++
fpe_mask = ekat::get_enabled_fpes();
```
before you start using it?

Reply:
It's on line 70 in this file.